pkg/kubernetes/util_test.go

```diff
@@ -56,7 +56,7 @@ var _ = Describe("Hostnames for a kubernetes service", func() {
 		})
 		It("Returns the service name from its hostname", func() {
 			hostname := fmt.Sprintf("%s:%d", service, tests.ServicePort)
-			Expect(GetServiceFromHostname(hostname)).To(Equal(fmt.Sprintf("%s:%d", service, tests.ServicePort)))
+			Expect(GetServiceFromHostname(hostname)).To(Equal(service))
```

Comment:
Should we add another test for the case where the hostname is of the form `service:port`

Reply:
Doesn't this line already cover the case of  `service:port`  ? So adding another test might not be needed